pkg/api/admin/openshiftcluster.go

```diff
@@ -166,6 +167,15 @@ const (
 	InstallPhaseRemoveBootstrap
 )
 
+// ArchitectureVersion represents an architecture version
+type ArchitectureVersion int
+
+// ArchitectureVersion constants
+const (
+	ArchitectureVersionV1 ArchitectureVersion = iota
+	ArchitectureVersionV2
+)
```

Comment:
Do we want to have to maintain this in multiple places?  Does an enum make it harder for us to do `if version < X` in our code, which we might need to do?